includes/API/V2/Route/Relationships.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace TenUp\ContentConnect\API\V2\Route;
+
+use TenUp\ContentConnect\API\V2\AbstractRoute;
+
+use function TenUp\ContentConnect\Helpers\get_post_to_post_relationships_by;
+use function TenUp\ContentConnect\Helpers\get_post_to_user_relationships_by;
+
+/**
+ * Class Relationships
+ *
+ * @package TenUp\ContentConnect\API\V2\Route
+ */
+class Relationships extends AbstractRoute {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function register_routes() {
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/relationships',
+			array(
+				'args' => array(
+					'rel_type' => array(
+						'description'       => __( 'The relationship type to filter relatioships by.', 'tenup-content-connect' ),
+						'type'              => 'string',
+						'default'           => 'post-to-post',
+						'sanitize_callback' => 'sanitize_text_field',
+						'validate_callback' => 'rest_validate_request_arg',
+						'enum'              => array( 'post-to-post', 'post-to-user' ),
+					),
+				),
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_items' ),
+					'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				),
+			),
+		);
+	}
+
+	/**
+	 * Retrieves a collection of relationships by type.
+	 *
+	 * @since 1.7.0
+	 *
+	 * @param  \WP_REST_Request $request Full details about the request.
+	 * @return \WP_REST_Response
+	 */
+	public function get_items( \WP_REST_Request $request ) {
+
+		$rel_type = $request->get_param( 'rel_type' );
+
+		$relationships = array();
+
+		switch ( $rel_type ) {
+			case 'post-to-post':
+				$relationships = get_post_to_post_relationships_by();
```

Comment:
It would be nice to be able to pass arguments to this function: `$field` and `$value` would let users filter this list to a specific post type.

Reply:
It's possible to filter relationships now that I added the `filter_by` and `filter_value` parameters to the endpoint, @benlk.